src/Components/Magazine_Block_Controller.php

```diff
@@ -14,44 +14,68 @@ class Magazine_Controller {
 	public function __construct( $block ) {
 		$this->block = (array) $block;
 	}
-	
-	public function get_title(): string {
-		$title = (string) get_field( Magazine_Block::TITLE );
-		
-		return strlen( $title ) < 1 ? '' : $title;
+
+	public function get_attributes(): string {
+		return wp_kses_data( get_block_wrapper_attributes([
+			'class' => implode(' ', [
+				'ucsc-magazine-block',
+				'alignfull',
+				'has-lightest-gray-background-color',
+				'has-global-padding',
+				'is-layout-constrained',
+			]),
+		]) );
 	}
 
-	public function get_overline(): string {
-		$overline = (string) get_field( Magazine_Block::OVERLINE );
+	public function get_title_line_1(): string {
+		$overline = (string) get_field( Magazine_Block::TITLE_LINE_1 );
 
 		return strlen( $overline ) < 1 ? '' : $overline;
 	}
 	
+	public function get_title_line_2(): string {
+		$title = (string) get_field( Magazine_Block::TITLE_LINE_2 );
+		
+		return strlen( $title ) < 1 ? '' : $title;
+	}
+	
+	public function get_subtitle(): string {
+		$subtitle = (string) get_field( Magazine_Block::SUBTITLE );
+		
+		return strlen( $subtitle ) < 1 ? '' : $subtitle;
+	}
+
 	public function get_magazines(): array {
 		$magazines = (array) get_field( Magazine_Block::ITEMS );
 		
 		if ( empty( $magazines ) ) {
 			return [];
 		}
 		
-		return $magazines;
+		return array_filter($magazines, static function ($magazine) {
+			return is_array( $magazine );
+		});
 	}
 	
-	public function make_tab_key( string $item, int $index ): string {
-		return sprintf( '%s-%s', sanitize_title( $item ), $index + 1 );
+	public function make_tab_key( array $magazine, int $index, string $element = '' ): string {
+		return sprintf( 
+			'%s-%s%s',
+			sanitize_title( $magazine[ Magazine_Block::ITEM_TITLE ] ),
+			$index + 1,
+			$element ? '__' . esc_html( $element ) : ''
+		);
 	}
 	
-	public function get_cta( array $magazine ): string {
+	public function get_cta( array $magazine ): array {
 		if ( empty( $magazine[ Magazine_Block::ITEM_CTA_FIELD ] ) || empty( $magazine[ Magazine_Block::ITEM_CTA_FIELD ]['title'] ) || empty( $magazine[ Magazine_Block::ITEM_CTA_FIELD ]['url'] ) ) {
-			return '';
+			return [];
 		}
 
-		return sprintf( 
-			'<a href="%s" target="%s">%s</a>', 
-			$magazine[ Magazine_Block::ITEM_CTA_FIELD ]['url'], 
-			$magazine[ Magazine_Block::ITEM_CTA_FIELD ]['target'] ?: '_self', 
-			$magazine[ Magazine_Block::ITEM_CTA_FIELD ]['title'] 
-		);
+		return [
+			'title'  => esc_html( $magazine[ Magazine_Block::ITEM_CTA_FIELD ]['title'] ),
+			'url'    => esc_attr( $magazine[ Magazine_Block::ITEM_CTA_FIELD ]['url'] ),
```

Comment:
you may cause an issue with esc here. If someone put a query link it can lost some special chats